Windows.bat

```diff
@@ -0,0 +1,74 @@
+::    limitations under the License.
+
+@echo off
+if "%1"=="a" (set automatic=a) else (if "%1"=="-a" (set automatic=a) else (set automatic=No))
+if "%2"=="retry" (set retry=retry) else (set retry=No)
+set script=%~f0
+echo Checking Conda installation...
+where conda
+if %errorlevel% == 0 (goto condainstalled)
+
+:installconda
+echo Checking existing conda installs not in path...
+set a="%APPDATA%\Microsoft\Windows\Start Menu\Programs\Anaconda3 (64-bit)\Anaconda Prompt (anaconda3).lnk"
+set b="%APPDATA%\Microsoft\Windows\Start Menu\Programs\Anaconda3 (64-bit)\Anaconda Prompt (miniconda3).lnk"
+set c="%APPDATA%\Microsoft\Windows\Start Menu\Programs\Anaconda3 (32-bit)\Anaconda Prompt (anaconda3).lnk"
+set d="%APPDATA%\Microsoft\Windows\Start Menu\Programs\Anaconda3 (32-bit)\Anaconda Prompt (miniconda3).lnk"
+if exist %a% if "%3" neq "a" if "%3" neq "b" if "%3" neq "c" if "%3" neq "d" call "cmd /c" %a% "& %script:"=% %automatic% %retry% a" & exit 0
+if exist %b% if "%3" neq "b" if "%3" neq "c" if "%3" neq "d" call "cmd /c" %b% "& %script:"=% %automatic% %retry% b" & exit 0
+if exist %c% if "%3" neq "c" if "%3" neq "d" call "cmd /c" %c% "& %script:"=% %automatic% %retry% c" & exit 0
+if exist %d% if "%3" neq "d" call "cmd /c" %d% "& %script:"=% %automatic% %retry% d" & exit 0
+
+echo Downloading miniconda3...
+if "%retry%"=="retry" (echo Conda still not found after installation, aborting & pause & exit /B 1)
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set OSBIT=32BIT || set OSBIT=64BIT
+if %OSBIT%==32BIT curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86.exe --output miniconda-installer.exe
+if %OSBIT%==64BIT curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe --output miniconda-installer.exe
+echo Installing miniconda3 (just use the default settings if you are unsure)...
+if "%automatic%"=="a" (start /wait miniconda-installer.exe /S /D=%USERPROFILE%\miniconda3) else (start /wait miniconda-installer.exe)
+del "miniconda-installer.exe"
+start /i /b %script% %automatic% retry & exit 0
+
+:condainstalled
+echo Conda installed
+call conda.bat config --add channels conda-forge
+call conda.bat activate mssenv
+if not %errorlevel% == 0 (
+    echo mssenv not found, creating...
+    call conda.bat create -n mssenv mamba -y
+    call conda.bat activate mssenv
+    if errorlevel 1 (echo Environment not found, aborting & pause & exit /B 1)
+)
+
+:envexists
+echo Installing mss...
+call mamba install mss python -y
+call mamba list -f mss| findstr /i /e "conda-forge"
+if not %errorlevel% == 0 (echo MSS was not successfully installed, aborting & pause & exit /B 1)
+
+echo Done! To start mss: Press the Windows button, type in "mss" and press enter.
+echo Alternatively:
+echo 1. Activate your conda environment with this command: "conda activate mssenv"
+echo 2. Start mss with this command: "mss"
+pause
```

Comment:
alternativly

press the Menu Key (between STRG and ALT), type mss into the search field, press enter

Reply:
![keyboard](https://user-images.githubusercontent.com/278728/133043533-6ea0f5bb-5536-427f-a909-98ec8ee4b69c.gif)

Reply:
Is `Press the Windows button, type in "mss" and press enter.` not descriptive enough?